src/main/java/org/takes/http/MainRemote.java

```
@@ -104,7 +106,9 @@ public void exec(final MainRemote.Script script) throws Exception {
                 )
             );
         } finally {
-            file.delete();
+            if (!file.delete()) {
```

@yegor256 ok let's throw an exception then